tests/server/test_loggers.py

```diff
@@ -54,6 +54,33 @@ def test_default_logger():
     assert isinstance(server_logger.logger.loggers[0].logger.loggers[0], PythonLogger)
 
 
+def test_data_logging_from_predefined():
+    server_config = ServerConfig(
+        endpoints=[
+            EndpointConfig(
+                task=task,
+                name=name,
+                model=stub,
+                add_predefined=[MetricFunctionConfig(func="sentiment_analysis")],
```

Comment:
Maybe I would also couple this test with the template logs, probably a good idea.